### PR TITLE
Added Inline Docs for Videopress.php

### PR DESCRIPTION
**Pull request by c3zh**
Added do_action info for modules/videopress/videopress.php

### CODE REVIEW

File: modules/videopress/videopress.php [hunk 1]
```diff
@@ -600,7 +600,13 @@ function enqueue_admin_scripts() {
 			'caps' => $caps,
 			'l10n' => $l10n,
 		) );
-
+		
+		/** 
+		 * Fires after VideoPress scripts are enqueued in the dashboard. 
+		 *
+		 * @Since 2.5.0
```
Comment (jeherve) [inline modules/videopress/videopress.php]: Could you make sure `@since` is all lowercase? Thank you!

File: modules/videopress/videopress.php [hunk 2]
```diff
@@ -600,7 +600,13 @@ function enqueue_admin_scripts() {
...
+		 */
+
```
Comment (jeherve) [inline modules/videopress/videopress.php]: I would remove that empty line here, it is not necessary.

**Comment by jeherve** [comment]
@c3zh It seems like you've added unrelated commits to this branch. You'll need to remove your changes from `functions.gallery.php`.

File: modules/videopress/videopress.php [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<?php
+ <?php
```
Comment (kraftbj) [inline modules/videopress/videopress.php]: This space should be removed. :)

Comment (c3zh) [inline modules/videopress/videopress.php]: I removed the space. Thanks!

**Comment by jeherve** [comment]
LGTM!

**Comment by c3zh** [comment]
WooHOOO!

**Pull request merged**